Lombiq.Analyzers/Docs/ConfiguringAnalyzers.md

```diff
@@ -20,6 +20,18 @@ This will completely disable code analysis packages. To also disable .NET SDK an
 dotnet_analyzer_diagnostic.category-Style.severity = none
 ```
 
+Note that MSBuild only loads in the closest _Directory.Build.props_ file to the project being built. So if you also used a _Directory.Build.props_ file to enable _Lombiq.Analyzers_ in a higher directory, it is now overridden. This can be a problem if you rely on `Lombiq.Analyzers` to set up compiler properties such as `<LangVersion>`. Put this into the _Directory.Build.props_ file instead:
```

Comment:
That wouldn't work if you include the project from NuGet package.

Reply:
What do you mean? If you do that then the package reference is either coming from a Directory file too, in which case it would work. Or is added directly to the csproj, but then you'd just remove the package, or use `<RunCodeAnalysis>false</RunCodeAnalysis>` too.